packages/soundux/soundux.pacscript

```diff
@@ -20,5 +21,13 @@ build() {
 }
 
 install() {
-  sudo make install DESTDIR=/usr/src/pacstall/soundux
+  sudo make install DESTDIR="$pkgdir"
+  # install binary symlink
+  sudo mkdir -p "$pkgdir/usr/bin/"
+  sudo ln -sf /opt/soundux/soundux "$pkgdir/usr/bin/soundux"
```

Comment:
Sorry for that weird approval, I did it by mistake.

So you are symlinking the binary from `/opt/` to the `pkgdir`?

Shouldn't this be the other way round? I mean the `/opt/soundux/soundux` is a non existent file

Reply:
With ln the first parameter is the location where it should point and the second one where the symlink should live

Reply:
Move the symlink to `postinst()`, then `stow` won't interfere with anything